webpack.config.js

```diff
@@ -30,7 +30,7 @@ const config = {
 		'admin-shortcodes': './includes/admin/shortcodes/admin-shortcodes.js',
 		'plugin-deactivation-survey': [ './assets/src/css/admin/plugin-deactivation-survey.scss', './assets/src/js/admin/plugin-deactivation-survey.js' ],
 		'admin-add-ons': [ './assets/src/js/admin/admin-add-ons.js' ],
-		'give-sequoia-theme': [ './src/Views/Form/Themes/Sequoia/assets/css/form.scss', './src/Views/Form/Themes/Sequoia//assets/js/form.js' ],
+		'give-sequoia-theme': [ './src/Views/Form-Themes/Sequoia/assets/css/form.scss', './src/Views/Form/Themes/Sequoia/assets/js/form.js' ],
```

Comment:
@henryholtgeerts form themes directory has been changed. it is `src/Views/Form/Theme`. Please fix the path here.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/a0f29ad69df4ee7d4745fa18995ea9e9c4acfb22